client.go

```diff
@@ -189,6 +189,8 @@ type Client interface {
 	GetVolumeGroupSnapshots(ctx context.Context) ([]VolumeGroup, error)
 	GetVolumeGroupSnapshotByName(ctx context.Context, snapName string) (VolumeGroup, error)
 	GetMaxVolumeSize(ctx context.Context) (int64, error)
+	ConfigureMetroVolume(ctx context.Context, volID string, config *MetroConfig) (resp MetroSessionID, err error)
```

Comment:
We could go with just 'id' as param name since function name specifies that the action is on a volume.

Reply:
make sense to me